apps/bot-discord/src/config.json

```diff
@@ -42,16 +42,16 @@
     "channelId": "1135563848586379264",
     "stickyMessage": {
       "title": "Sticky notice",
-      "description": "In this channel, you can get assistance with your questions, issues, or concerns. Before continuing, please continue to read this sticky message.",
+      "description": "",
```

Comment:
I believe you can simply remove it. This is a native embed object from Discord.js.

```suggestion
```